### DISCUSSION TITLE
Cannot connect ELM327 PIC18F25K80 to ddt4all

### DISCUSSION THREAD

**Discussion by RiskyBussiness** [cedricp/ddt4all#973] [Q&A]
Hello, today I got my ELM327 device, finally. It's a wifi variant (ELM327 V1.5 OBD2 PIC18F25K80 Wifi exactly). I can connect it to my laptop wifi but it is not listed on ddt4all ELM Port Selection screen. I am absolutely lost, don't even know where to start.

**Comment by Furtif** [comment]
🙄

**Comment by KarelSvo** [comment]
@RiskyBussiness 
To select the Wifi address you have to click on the Wifi symbol.
If you use the ddt4all Python 3 version, the WiFi connection is disturbed.
There is a bugfix for the problem, but it has not been committed to the master branch.
Alternatively, I can give you an installation file for a stable Python 2 version.

**Reply by enesokyay** [reply]
I'm having the same problem now. I can log in via phone, but when I pass the pc, the application does not connect to the vehicle. I made the wifi settings manually. I use a Mac m1 silicon processor
